File: compile.js
const { execSync } = require('child_process');
const path = require('path');

const dir = path.join('C:\\Users\\awepu\\OneDrive\\Desktop', 'Новая папка (5)');
process.chdir(dir);

try {
    const output = execSync('gradlew.bat compileClientJava', { encoding: 'utf-8', stdio: 'inherit' });
    console.log(output);
} catch (e) {
    console.error('Compilation failed:', e.message);
    process.exit(1);
}

File: compile.py
import subprocess
import os

os.chdir(r"C:\Users\awepu\OneDrive\Desktop\Новая папка (5)")
result = subprocess.run([r"gradlew.bat", "compileClientJava"], capture_output=True, text=True)
print(result.stdout)
print(result.stderr)
print(f"Return code: {result.returncode}")
